sustainability-best-practices/SUMMARY.md

```diff
@@ -9,6 +9,11 @@
 * [Architecture](infrastructure/architecture.md)
 * [Security and Bots](infrastructure/security-and-bots.md)
 
+## Telemetry
+
+* [About telemetry](telemetry/README.md)
```

Comment:
to match with the rest
```suggestion
* [Telemetry Overview](telemetry/README.md)
```

Reply:
@eshanrnh there will be new articles once a year, as the list might get long later on, would it make better sense to move this at the end of the summary instead? 🤔